### ISSUE TITLE
Maximum capacity of an array (in elements) is 8191

### ISSUE THREAD

**Issue by AlqanbarRami** [gotenberg/gotenberg#847]
I am trying to convert some files into PDF/A-1b format and am receiving a response code of 200. However, when I check the file in VeraPDF, I see that the file has not been converted correctly. I am getting the following error details:

Error details:
The maximum capacity of an array (8191) is exceeded.

Object type: CosArray
Test condition: size <= 8191
Specification: ISO 19005-1:2005
Levels: A, B
Additional references:
PDF Reference 1.4, Table C.1

**Comment by gulien** [comment]
Hello @AlqanbarRami,

It has been converted successfully… for the most part. If it’s not 100% valid, it’s an issue within LibreOffice, and, good news, it has already been reported in tjeir big tracker: https://bugs.documentfoundation.org/show_bug.cgi?id=160196. Now we have to wait for a fix.

**Comment by AlqanbarRami** [comment]
Hello again, @gulien!

Thank you for your response. I understand now that the file has indeed been converted, but it isn't compliant with the PDF/A-1b standard. Interestingly, when I attempted to convert the same file to PDF/A-3b, it converted successfully and was compliant with the PDF/A-3b standard. It appears that the issue is isolated to the conversion process for PDF/A-1b and PDF/A-2b.

I've uploaded the file due to its size. It's worth noting that this issue isn't isolated to just this one file; I have 2-3 other files encountering the same problem.
https://easyupload.io/z4i0l7
